### DISCUSSION TITLE
Add lateral supports for high object

### DISCUSSION THREAD

**Discussion by philipgo** [OrcaSlicer/OrcaSlicer#4314] [Q&A] [answered]
Hello,

is there a way to automatically add lateral supports for high objects so they do not move while printing? Cura does this, but I would prefer to use OrcaSlicer.

Thank you!
![grafik](https://github.com/SoftFever/OrcaSlicer/assets/29184757/c06df1cd-6fb9-471c-b966-5a043e5562db)

**Comment by foreachthing** [comment, marked as answer]
In your case, it might be enough to change the "Threshold Angle" closer or equal to 90°.

Or, use "Paint on Support":
![image](https://github.com/SoftFever/OrcaSlicer/assets/10420187/05a2f0bf-9711-4958-9fc7-86926169d041)![image](https://github.com/SoftFever/OrcaSlicer/assets/10420187/2b65d2ca-25b2-4694-9ba5-1da69ac07707)

And Settings like these:
![image](https://github.com/SoftFever/OrcaSlicer/assets/10420187/e9979a40-6e89-4060-8e8f-1b54fcd08162)

**Reply by philipgo** [reply]
Thank you! Both solutions work :)

**Reply by philipgo** [reply]
Another question:
How would you print this? Even when I paint supports on the vertical walls, none are created.
![grafik](https://github.com/SoftFever/OrcaSlicer/assets/29184757/00fa00e4-cfb1-47f8-96df-97a946f80a38)
